README.md

```diff
@@ -87,6 +87,12 @@ Options:
                                   specified multiple times.
   --config FILE                   Read configuration from specified file.
                                   [default: tartufo.toml]
+  --target-config/--no-target-config BOOL
```

Comment:
```suggestion
  --target-config/--no-target-config
```

I think this better aligns with the usage documentation for eg `--scan-filenames / --no-scan-filenames`